CharlieBackend.Core/DTO/Dashboard/StudentGroupResults/AverageStudentGroupMarkDto.cs

```diff
@@ -6,7 +6,7 @@ namespace CharlieBackend.Core.DTO.Dashboard
 {
     public class AverageStudentGroupMarkDto
     {
-        public long CourseId { get; set; }
+        public long? CourseId { get; set; }
```

Comment:
Lets try to avoid types changes in DTOs, to avoid mistakes due to existing logic in services and validations

Reply:
done